model/app/installer.go

```diff
@@ -409,7 +410,22 @@ func (i *Installer) update() error {
 	// verifyPermissions flag is activated (for non manual updates for example),
 	// we cancel out the update and mark the UpdateAvailable field of the
 	// application instead of actually updating.
-	if makeUpdate && !isPlatformApp(oldManifest, i.context) {
+	if build.IsDevRelease() {
+		// If we are in dev release we want to automatically accept any permissions set
```

Comment:
I'm not sure I agree with this change. It can be useful for developers to test an upgrade with a permissions change.

Reply:
You still can if you don't use the `--dev` flag. All depends of the meaning you want to give to the flag `--dev`. For me I understood it as the flag setup when you want to dev locally and you need to remove some protections allowing hot-reload, removing some check or stuff like that.